src/events/ready.ts

```diff
@@ -19,7 +19,7 @@ export default TypedEvent({
         const commandArr: object[] = [];
 
         for await (const file of commandFiles) {
-            const command = (await import(file)).command as IBotCommand;
+            const command = (await import(file)).default as IBotCommand;
```

Comment:
Reason and impacts of this? Did you run tests to ensure it doesn't break anything?

Reply:
> Reason and impacts of this? Did you run tests to ensure it doesn't break anything?

I think this is because of the new pull i made at https://github.com/conaticus/boolean/commit/c99707beb67eb90a2fbd3be8439df294dec02839 and https://github.com/conaticus/boolean/commit/b282495dc99e3cb996c4a7e4352b804221a30439

`const command = (await import(file)) as IBotCommand;`
not sure if it works. but since its 'exports default' this should work.